pkgs/tools/graphics/pdftag/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, pkgconfig, meson, vala, ninja
+, gtk3, poppler, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "pdftag";
+  name = "${pname}-${version}";
+  version = "1.0.4";
+
+  src = fetchFromGitHub {
+    owner = "arrufat";
+    repo = pname;
+    rev = version;
+    sha256 = "17zk42h0n33b4p8fqlq2riqwcdi8y9m5n0ccydnk6a4x8rli97b3";
+  };
+
+  nativeBuildInputs = [ pkgconfig meson ninja gtk3 poppler wrapGAppsHook ];
```

Comment:
`gtk3` and `poppler` should go to `buildInputs`, since they need to match the user’s architecture.